libs/net/dal/Services/ContentService.cs

```diff
@@ -166,187 +170,134 @@ public IPaged<Content> FindWithDatabase(ContentFilter filter, bool asNoTracking
     /// <returns>A page of content items that match the filter.</returns>
     public async Task<IPaged<Content>> FindWithElasticsearchAsync(ContentFilter filter)
     {
+        var sourceQueries = new List<Func<QueryContainerDescriptor<Content>, QueryContainer>>();
```

Comment:
@CelloLiu There were a lot of bugs with elasticsearch when filtering on a number of these criteria.  I fixed the ones I was testing.